index.html

```diff
@@ -438,13 +451,11 @@ <h3>
                 </li>
                 <li>If |url| is failure, return false.
                 </li>
-                <li>
-                  <p>
-                    If |url| is a URL the user agent deems potentially hostile
-                    (e.g., "file:") or wouldn't make sense to outside the scope
-                    of the document (e.g., "blob:"), return false.
-                  </p>
-                  <aside class="issue" data-number="178"></aside>
+                <li>If the |url|'s [=URL/scheme=] is a [=local scheme=], or
```

Comment:
I know this check is kinda redundant, because they are not in the the "sharable scheme" list, but I wanted to be explicit in banning them.